usr/local/www/services_unbound.php

```diff
@@ -325,6 +338,33 @@ function show_advanced_dns() {
 								</td>
 							</tr>
 							<tr>
+								<td width="22%" valign="top" class="vncellreq"><?=gettext("System Domain Local-Zone Type"); ?></td>
+								<td width="78%" class="vtable">
+									<?php
+										$unbound_local_zone_types = get_posible_unbound_local_zone_types(false);
+										$size=count($unbound_local_zone_types);
```

Comment:
We usually put space around equals signs on ordinary lines like this. It looks inconsistent with the line above :)
$size = count(...